c2corg_ui/templates/document/route_card.html

```diff
@@ -1,6 +1,6 @@
 <a>
   <span class="list-item-title">{{route['locales'][0]['title']}}</span>
-  <span class="list-item-lang"> {{route['locales'][0]['lang']}}</span><br>
+  <span class="list-item-lang"> ({{route['locales'][0]['lang']}})</span><br>
```

Comment:
The lang info is actually needed only when the document's lang is not the current interface language.
Could we show it only in that case? For instance with
```ng-show="{{route['locales'][0]['lang']}} != <interface lang>"```
?